portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -72,4 +72,26 @@ private String getParameter(HttpServletRequest request, String name) {
     }
     return value;
   }
+
+  private int getMaxCommentsToDisplay(HttpServletRequest request) {
+    String maxCommentsToDisplayString = "3"; //temporary hardcoded value, will get replaced with user input
+
+    // Convert the input to an int.
+    int maxCommentsToDisplay;
+    try {
+      maxCommentsToDisplay = Integer.parseInt(maxCommentsToDisplayString);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + maxCommentsToDisplayString);
+      return -1;
+    }
+
+    // Check that the input is greater than 0
+    if (maxCommentsToDisplay < 0) {
+      System.err.println("Max comments to display cannot be negative: " + maxCommentsToDisplayString);
+      return -1;
```

Comment:
Instead of using a sentinel value here, I would just throw an exception here, and handle when it is called.

Reply:
Threw an exception, caught in doGet. I will have to do some more thinking about how/where to properly handle the exception.

Reply:
I have changed the html page (future PR) so that it prevents users from entering in negative numbers or non-number values.